code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/ldap/interfaces/LDAPConfig.java

```diff
@@ -44,4 +48,6 @@
 
     long getConnectionTimeout();
 
+    List<Identity> getAllowedIdentities();
```

Comment:
This one is used by the caller(UI) of the /v1/ldapconfig API to get the list of "allowed" Identities that have been set for this auth provider.
Internally the auth provider framework, stores the List<Identity> as a comma/hash separated string of id's in the settings table.